src/dataModels/SummarySubgrid.js

```diff
@@ -28,11 +28,11 @@ SummarySubgrid.prototype = {
     hasOwnData: true, // do not call setData implicitly
 
     getRowCount: function() {
-        return this.behavior.dataModel.getRowCount() && this.getData().length;
+        return this.getData().length;
     },
 
     getData: function() {
-        return this.behavior.dataModel.dataSource.getGrandTotals() || this.data;
+        return this.data;
```

Comment:
Don't worry about this for now; I'll do it with my subgrids PR... I also fixed `setValue` which is broke (not in use; never tested).